cmd/grr/workflow.go

```diff
@@ -104,3 +124,35 @@ func watchCmd() *cli.Command {
 	}
 	return cmd
 }
+
+func exportCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "export <jsonnet-file> <dashboard-dir>",
+		Short: "render Jsonnet and save to a directory",
+	}
+	targets := cmd.Flags().StringSliceP("target", "t", nil, "dashboards to target")
+	cmd.Run = func(cmd *cli.Command, args []string) error {
+		if len(args) == 0 || len(args) > 2 {
+			return fmt.Errorf("Incorrect arguments, expected: <jsonnet-file> [<dashboard-dir>]")
+		}
+		jsonnetFile := args[0]
+		dashboardDir := ""
+		if len(args) == 2 {
+			dashboardDir = args[1]
+		}
```

Comment:
go-clix/cli has `ArgsExact` for this! https://github.com/go-clix/cli#positional-arguments...

```suggestion
		Args:  cli.ArgsExact(2),
	}
	targets := cmd.Flags().StringSliceP("target", "t", nil, "dashboards to target")
	cmd.Run = func(cmd *cli.Command, args []string) error {
		jsonnetFile := args[0]
		dashboardDir := args[1]
```